src/test/java/com/github/dariobalinzo/filter/WhitelistFilterTest.java

```diff
@@ -49,12 +52,36 @@ public void shouldConvertSimpleSchema() {
                 "version"
         ).collect(Collectors.toCollection(HashSet::new));
         WhitelistFilter whitelistFilter = new WhitelistFilter(filterValues);
-        Map<String, Object> filteredElasticDocument = whitelistFilter.filter(elasticDocument);
+        whitelistFilter.filter(elasticDocument);
 
         //then
-        assert (filteredElasticDocument.toString()).equals("{surname=search, name=elastic, version=7}");
```

Comment:
it is better to user junit Assert class to have a description when the test fails